msteams-platform/toolkit/debug-local.md

```diff
@@ -0,0 +1,292 @@
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/hot-reload.png" alt-text="hot-reload for source codes":::
+
+### Stop debugging
+
+When you complete local debug, you can select **Stop** or **Disconnect** from the floating debugging toolbar to stop all debug sessions and terminate tasks. The following image shows the stop debug action:
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/stop-debug.png" alt-text="stop debugging":::
+
+## Debug locally using Teams Toolkit
+
+### 1. Debug your app on Windows and macOS operating systems
+
+Complete the following steps to debug your app after creating a new app using the Teams Toolkit:
+
+<br>
+
+<details>
+<summary><b>Windows</b></summary>
+
+1. Select **Debug (Edge)** or **Debug (Chrome)** from the **Run and Debug** in the activity bar.
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/debug-run.png" alt-text="Browser option" border="false":::
+
+1. Select **Start Debugging (F5)** or  **Run** :::image type="content" source="../assets/images/teams-toolkit-v2/debug/run.png" alt-text="Start to debug" border="false"::: to run your Teams :::image type="content" source="../assets/images/teams-toolkit-v2/debug/teams-icon.png" alt-text="Teams icon" border="false":::  app in debug mode.
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/start-debugging.png" alt-text="Start debugging" border="false":::
+
+3. Select **Sign in** to :::image type="icon" source="../assets/icons/microsoft-icon.png" border="false"::: Microsoft 365 account.
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/microsoft365-signin.png" alt-text="Sign in" border="true":::
+
+
+   > [!TIP]
+   > You can select **Read more** to learn about :::image type="icon" source="../assets/icons/microsoft-icon.png" border="false"::: Microsoft 365 Developer Program. Your default web browser opens to let you sign in to your :::image type="icon" source="../assets/icons/microsoft-icon.png" border="false"::: Microsoft 365 account using your credentials.
+
+4. Select **Install** for installing the development certificate for localhost.
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/debug/install-certificate.png" alt-text="certificate" border="true":::
+
+   > [!TIP]
+   > You can select **Learn More** to know about the development certificate.
+
+5. Select **Yes** if the following dialog appears:
+
+    :::image type="content" source="../assets/images/teams-toolkit-v2/debug/development-certificate.png" alt-text="certification authority" border="true":::
+
+:::image type="icon" source="../assets/icons/sidebar-icon.png" border="false"::: Toolkit launches a new Edge or Chrome browser instance depending on your selection and opens a web page to load Teams client. 
+
+</details>
+
+
+<details>
+<summary><b>macOS**</b></summary>
```

Comment:
Can we merge Windows and macOS, since most of them are the same.

Reply:
As they are two seperate operating systems, if we merge them together, it affects the flow of the page, also create ambiguity and confusion for the target audience.